src/Room/Room.js

```diff
@@ -30,7 +30,7 @@ const Room = ({ roomName, room, handleLogout }) => {
   const [participants, setParticipants] = useState([]);
   const [vid, setVid] = useState(false);
   const [mic, setMic] = useState(false);
-  const isYoutube = roomName.includes("-YT-") ? true : false;
+  const [isYoutube, setIsYoutube] = useState(false);
```

Comment:
if you change this to true, you can see the Youtube room layout

Reply:
does the youtube work for you?

Reply:
actually, its an issue with the place we're pulling Youtube Metadata from in Search.js